### ISSUE TITLE
BACKTRACE 22eacc ttf_draw_string_raw_ttf

### ISSUE THREAD

**Issue by IntelOrca** [OpenRCT2/OpenRCT2#8227]
<table>
    <tr>
        <td><img src='https://backtrace.io/images/icon.png' width=30 height=30 /></td>
        <td><a href='https://openrct2.sp.backtrace.io/dashboard/openrct2/project/openrct2/group/22eacc05afd5078d99b5bb338480713955349390f2adaf4847066412d0a4cb53?_t=W3sibmFtZSI6IndvcmtmbG93In1d'><b>New unique error: [openrct2]</b></a><br /><b>1</b> occurrence(s) <b></b> since (Mon Nov 05 2018 10:50:26)<br /></td>
    </tr>
</table>

<table>
<tr><td><strong>Application</strong></td><td>openrct2.exe</td></tr>

<tr><td><strong>Attributes</strong></td><td><strong>commit:</strong> 3aeb87b: <em>1</em><br></td></tr>
<tr><td><strong>Classifiers</strong></td><td><strong>invalid-write</strong> </td></tr>
<tr><td><strong>Frames</strong></td><td><code>ttf_draw_string_raw_ttf -> ttf_process_string -> ttf_draw_string -> DrawText -> DrawTextCompat -> window_editor_bottom_toolbar_paint -> window_draw -> window_draw_all -> OpenRCT2::Drawing::X8DrawingEngine::DrawAllDirtyBlocks -> OpenRCT2::Drawing::X8DrawingEngine::PaintWindows -> OpenRCT2::Paint::Painter::Paint -> OpenRCT2::Context::RunGameLoop -> OpenRCT2::Context::Launch -> OpenRCT2::Context::RunOpenRCT2 -> NormalisedMain -> LaunchOpenRCT2 -> wmain</code></td></tr>
</table>

**Comment by janisozaur** [comment]
```
dump_path: /var/coronerd//openrct2/openrct2/_objects/0000/tx.008E.dmp outfile: -
Classifier: invalid-write
Crash info:
	Application: openrct2.exe
	Crashed YES
	reason: EXCEPTION_ACCESS_VIOLATION_WRITE
	address: 0x6d910fc
Memory Regions:
Sysinfo:
	arch: x86 GenuineIntel family 6 model 23 stepping 10
	cpu_count: 2
	os: Windows NT windows 10.0.14393 
Modules:
	0x11e0000 - 0x1209000	openrct2.exe 
	0xf540000 - 0x103a0000	openrct2.dll 
	0x50c00000 - 0x50c7c000	AudioSes.dll 6.2.14393.2214
	0x5f560000 - 0x5f571000	mskeyprotect.dll 6.2.14393.0
	0x5fb60000 - 0x5fb7b000	ncryptsslp.dll 6.2.14393.0
	0x670f0000 - 0x6712d000	icm32.dll 6.2.14393.953
	0x67c00000 - 0x67c0a000	secur32.dll 6.2.14393.0
	0x694b0000 - 0x694b8000	rasadhlp.dll 6.2.14393.0
	0x6a560000 - 0x6a585000	dbgcore.dll 6.2.14321.1024
	0x6a610000 - 0x6a755000	dbghelp.dll 6.2.14321.1024
	0x6a940000 - 0x6aba6000	wininet.dll 11.0.14393.2214
	0x6bd00000 - 0x6be68000	dui70.dll 6.2.14393.0
	0x6be70000 - 0x6bf84000	IMETIP.DLL 6.2.14393.2214
	0x6bfe0000 - 0x6c09d000	imkrtip.dll 6.2.14393.447
	0x6caf0000 - 0x6caf8000	IconCodecService.dll 6.2.14393.0
	0x6cb40000 - 0x6cb6f000	IMJKAPI.DLL 6.2.14393.2214
	0x6cb90000 - 0x6cbbd000	imkrapi.dll 6.2.14393.2214
	0x6cd30000 - 0x6cd79000	FWPUCLNT.DLL 6.2.14393.0
	0x6d0e0000 - 0x6d134000	oleacc.dll 7.2.14393.2156
	0x6dbc0000 - 0x6ddca000	comctl32.dll 6.10.14393.953
	0x6dfd0000 - 0x6e055000	mscms.dll 6.2.14393.953
	0x6e820000 - 0x6e878000	MMDevAPI.dll 6.2.14393.1198
	0x70060000 - 0x70068000	version.dll 6.2.14393.0
	0x711e0000 - 0x711e9000	avrt.dll 6.2.14393.0
	0x71230000 - 0x712ff000	WinTypes.dll 6.2.14393.2214
	0x71a50000 - 0x71bbe000	WindowsCodecs.dll 6.2.14393.2156
	0x71cb0000 - 0x71dff000	propsys.dll 7.0.14393.0
	0x71f50000 - 0x71f6f000	dwmapi.dll 6.2.14393.206
	0x72140000 - 0x72163000	WINMMBASE.dll 6.2.14393.0
	0x72480000 - 0x724a4000	winmm.dll 6.2.14393.0
	0x726c0000 - 0x7273a000	uxtheme.dll 6.2.14393.0
	0x72b10000 - 0x72b32000	devobj.dll 6.2.14393.0
	0x72b40000 - 0x72b68000	ntmarta.dll 6.2.14393.1378
	0x73060000 - 0x730c4000	schannel.dll 6.2.14393.2214
	0x73110000 - 0x7313f000	rsaenh.dll 6.2.14393.0
	0x73240000 - 0x7326f000	IPHLPAPI.DLL 6.2.14393.953
	0x73270000 - 0x732ed000	dnsapi.dll 6.2.14393.2155
	0x73300000 - 0x7331a000	userenv.dll 6.2.14393.0
	0x73450000 - 0x734a0000	mswsock.dll 6.2.14393.0 (Warning no symbol, mswsock.pdb CF53E7EAE83E451CB1C0243042BD32821)
	0x735c0000 - 0x735ca000	CRYPTBASE.dll 6.2.14393.0
	0x735d0000 - 0x735e3000	cryptsp.dll 6.2.14393.0
	0x736a0000 - 0x736cb000	ntasn1.dll 6.2.14393.0
	0x736d0000 - 0x736f0000	ncrypt.dll 6.2.14393.0
	0x737b0000 - 0x737d3000	sspicli.dll 6.2.14393.1770
	0x739b0000 - 0x739cd000	bcrypt.dll 6.2.14393.576
	0x73a50000 - 0x73a5e000	msasn1.dll 6.2.14393.0
	0x73a60000 - 0x73a6f000	profapi.dll 6.2.14393.0
	0x73a70000 - 0x73ab5000	powrprof.dll 6.2.14393.0
	0x73ac0000 - 0x73acd000	kernel.appcore.dll 6.2.14393.0
	0x73ad0000 - 0x73b4b000	msvcp_win.dll 6.2.14393.2097
	0x73b50000 - 0x73b69000	win32u.dll 6.2.14393.51
	0x73b70000 - 0x73c50000	ucrtbase.dll 6.2.14393.2097
	0x73c50000 - 0x73da9000	gdi32full.dll 6.2.14393.1770
	0x73db0000 - 0x7431e000	windows.storage.dll 6.2.14393.2214
	0x74410000 - 0x7458d000	crypt32.dll 6.2.14393.2214
	0x74590000 - 0x745ea000	bcryptPrimitives.dll 6.2.14393.2155
	0x745f0000 - 0x74797000	KERNELBASE.dll 6.2.14393.2189
	0x747a0000 - 0x747d6000	cfgmgr32.dll 6.2.14393.0
	0x747e0000 - 0x74864000	clbcatq.dll 2001.12.10941.16384
	0x74880000 - 0x748f7000	advapi32.dll 6.2.14393.2007
	0x74900000 - 0x749e5000	comdlg32.dll 6.2.14393.447
	0x74b60000 - 0x74bc3000	ws2_32.dll 6.2.14393.206
	0x74bd0000 - 0x74c58000	SHCore.dll 6.2.14393.2097
	0x74c60000 - 0x76038000	shell32.dll 6.2.14393.2214
	0x76040000 - 0x7606b000	gdi32.dll 6.2.14393.2155
	0x76070000 - 0x7612e000	msvcrt.dll 7.0.14393.0
	0x76130000 - 0x761f3000	rpcrt4.dll 6.2.14393.2155
	0x76200000 - 0x76411000	combase.dll 6.2.14393.2214
	0x76420000 - 0x76466000	shlwapi.dll 6.2.14393.0
	0x76470000 - 0x765a5000	msctf.dll 6.2.14393.2035
	0x765b0000 - 0x7669c000	ole32.dll 6.2.14393.2007
	0x76b30000 - 0x76c79000	user32.dll 6.2.14393.1770
	0x76ce0000 - 0x76d21000	sechost.dll 6.2.14393.1613
	0x76d30000 - 0x76dc6000	kernel32.dll 6.2.14393.2214
	0x76de0000 - 0x76e06000	imm32.dll 6.2.14393.0
	0x76e10000 - 0x76ea4000	oleaut32.dll 6.2.14393.1378
	0x76f10000 - 0x76f17000	nsi.dll 6.2.14393.0
	0x76f20000 - 0x770a6000	ntdll.dll 6.2.14393.2214
Threads:
	thread-0 tid=8076 faulted
	[0] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!ttf_draw_string_raw_ttf(rct_drawpixelinfo *,char const *,text_draw_info *) [c:\projects\openrct2-ject9\src\openrct2\drawing\drawing.string.cpp : 594 + 0x3]
		ip=0xf6254ee sp=0x108f580 bp=0x108f5bc 
	[1] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!ttf_process_string(rct_drawpixelinfo *,char const *,text_draw_info *) [c:\projects\openrct2-ject9\src\openrct2\drawing\drawing.string.cpp : 841 + 0x12]
		ip=0xf6259fe sp=0x108f5c4 bp=0x108f7f4 
	[2] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!ttf_draw_string(rct_drawpixelinfo *,char const *,int,int,int) [c:\projects\openrct2-ject9\src\openrct2\drawing\drawing.string.cpp : 931 + 0xe]
		ip=0xf625c08 sp=0x108f7fc bp=0x108f838 
	[3] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!DrawText(rct_drawpixelinfo *,int,int,TextPaint *,char const *) [c:\projects\openrct2-ject9\src\openrct2\drawing\text.cpp : 96 + 0x13]
		ip=0xf623c6b sp=0x108f840 bp=0x108f888 
	[4] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!DrawTextCompat(rct_drawpixelinfo *,int,int,unsigned short,void *,unsigned char,TextAlignment,bool) [c:\projects\openrct2-ject9\src\openrct2\drawing\text.cpp : 124 + 0x62]
		ip=0xf623dc7 sp=0x108f890 bp=0x108faa4 
	[5] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!window_editor_bottom_toolbar_paint(rct_window *,rct_drawpixelinfo *) [c:\projects\openrct2-ject9\src\openrct2-ui\windows\editorbottomtoolbar.cpp : 465 + 0x8]
		ip=0xf5894b1 sp=0x108faac bp=0x108fae0 
	[6] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!window_draw(rct_drawpixelinfo *,rct_window *,int,int,int,int) [c:\projects\openrct2-ject9\src\openrct2\interface\window.cpp : 1160 + 0x14e]
		ip=0xf5f1750 sp=0x108fae8 bp=0x108fb34 
	[7] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!window_draw_all(rct_drawpixelinfo *,short,short,short,short) [c:\projects\openrct2-ject9\src\openrct2\interface\window.cpp : 2119 + 0x15]
		ip=0xf5f27ff sp=0x108fb3c bp=0x108fb78 
	[8] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Drawing::X8DrawingEngine::DrawAllDirtyBlocks() [c:\projects\openrct2-ject9\src\openrct2\drawing\x8drawingengine.cpp : 428 + 0x9b]
		ip=0xf622d0d sp=0x108fb80 bp=0x108fbc4 
	[9] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Drawing::X8DrawingEngine::PaintWindows() [c:\projects\openrct2-ject9\src\openrct2\drawing\x8drawingengine.cpp : 224 + 0x7]
		ip=0xf6227d7 sp=0x108fbcc bp=0x108fd6c 
	[10] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Paint::Painter::Paint(OpenRCT2::Drawing::IDrawingEngine &) [c:\projects\openrct2-ject9\src\openrct2\paint\painter.cpp : 44 + 0x7]
		ip=0xf6b74a9 sp=0x108fbd8 bp=0x108fd6c 
	[11] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Context::RunGameLoop() [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 801 + 0xe9]
		ip=0xf5eab3c sp=0x108fd74 bp=0x108fd90 
	[12] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Context::Launch() [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 773 + 0x8]
		ip=0xf5ea9c7 sp=0x108fd98 bp=0x108fdf8 
	[13] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!OpenRCT2::Context::RunOpenRCT2(int,char const * *) [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 213 + 0x7]
		ip=0xf5e8c6a sp=0x108fe00 bp=0x108fe0c 
	[14] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!NormalisedMain(int,char const * *) [c:\projects\openrct2-ject9\src\openrct2-ui\ui.cpp : 60 + 0x13]
		ip=0xf563da7 sp=0x108fe14 bp=0x108fe84 
	[15] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!LaunchOpenRCT2(int,wchar_t * *) [c:\projects\openrct2-ject9\src\openrct2-dll\openrct2-dll.cpp : 46 + 0x9]
		ip=0xf5638c2 sp=0x108fe8c bp=0x108fea8 
	[16] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.exe!wmain(int,wchar_t * *,wchar_t *) [c:\projects\openrct2-ject9\src\openrct2-win\openrct2-win.cpp : 27 + 0xc]
		ip=0x11e100f sp=0x108feb0 bp=0x108feb8 
	[17] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.exe!__scrt_common_main_seh() [f:\dd\vctools\crt\vcstartup\src\startup\exe_common.inl : 288 + 0x1c]
		ip=0x11e11d3 sp=0x108fec0 bp=0x108ff00 
	[18] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0x108ff08 bp=0x108ff14 
	[19] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0x108ff1c bp=0x108ff5c 
	[20] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0x108ff64 bp=0x108ff6c 
	thread-1 tid=1272 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x130f744 bp=0x130f95c 
	[1] C:\Windows\System32\ntdll.dll!_ZwWaitForWorkViaWorkerFactory@20 + 0xa
		ip=0x76fb11ca sp=0x130f748 bp=0x130f95c 
	[2] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x2a6
		ip=0x76f69976 sp=0x130f74c bp=0x130f95c 
	[3] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0x130f964 bp=0x130f970 
	[4] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0x130f978 bp=0x130f9b8 
	[5] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0x130f9c0 bp=0x130f9c8 
	thread-2 tid=1572 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x16ff954 bp=0x16ffb6c 
	[1] C:\Windows\System32\ntdll.dll!_ZwWaitForWorkViaWorkerFactory@20 + 0xa
		ip=0x76fb11ca sp=0x16ff958 bp=0x16ffb6c 
	[2] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x2a6
		ip=0x76f69976 sp=0x16ff95c bp=0x16ffb6c 
	[3] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0x16ffb74 bp=0x16ffb80 
	[4] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0x16ffb88 bp=0x16ffbc8 
	[5] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0x16ffbd0 bp=0x16ffbd8 
	thread-3 tid=8168 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x29ff7ac bp=0x29ff9c4 
	[1] C:\Windows\System32\ntdll.dll!_ZwWaitForWorkViaWorkerFactory@20 + 0xa
		ip=0x76fb11ca sp=0x29ff7b0 bp=0x29ff9c4 
	[2] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x2a6
		ip=0x76f69976 sp=0x29ff7b4 bp=0x29ff9c4 
	[3] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0x29ff9cc bp=0x29ff9d8 
	[4] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0x29ff9e0 bp=0x29ffa20 
	[5] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0x29ffa28 bp=0x29ffa30 
	thread-4 tid=1460 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x32bf1f8 bp=0x32bf208 
	[1] C:\Windows\System32\KERNELBASE.dll!_OpenEventA@12 + 0x60
		ip=0x746b28a0 sp=0x32bf1fc bp=0x32bf208 
	thread-5 tid=8064 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x33bf910 bp=0x33bf950 
	[1] C:\Windows\System32\ntdll.dll!_NtWaitForAlertByThreadId@8 + 0xa
		ip=0x76fb128a sp=0x33bf914 bp=0x33bf950 
	[2] C:\Windows\System32\ntdll.dll!RtlSleepConditionVariableSRW + 0xe0
		ip=0x76fa8cd0 sp=0x33bf918 bp=0x33bf950 
	[3] C:\Windows\System32\KERNELBASE.dll!_SleepConditionVariableSRW@16 + 0x23
		ip=0x746fc5f3 sp=0x33bf958 bp=0x33bf974 
	[4] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!__crtSleepConditionVariableSRW(_RTL_CONDITION_VARIABLE *,_RTL_SRWLOCK *,unsigned long,unsigned long) [f:\dd\vctools\crt\crtw32\misc\winapisupp.cpp : 657 + 0x2]
		ip=0xf934189 sp=0x33bf97c bp=0x33bf990 
	[5] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!Concurrency::details::stl_condition_variable_win7::wait_for(Concurrency::details::stl_critical_section_interface *,unsigned int) [f:\dd\vctools\crt\crtw32\stdcpp\thr\primitives.h : 216 + 0x15]
		ip=0xfaa0700 sp=0x33bf998 bp=0x33bf9a8 
	[6] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!do_wait(_Cnd_internal_imp_t *,_Mtx_internal_imp_t *,xtime const *) [f:\dd\vctools\crt\crtw32\stdcpp\thr\cond.c : 82 + 0x5]
		ip=0xfaa05fe sp=0x33bf9b0 bp=0x33bf9e4 
	[7] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!std::_LaunchPad<std::unique_ptr<std::tuple<<lambda_b1de77b2e28beeaf6256a3f41ee36302> >,std::default_delete<std::tuple<<lambda_b1de77b2e28beeaf6256a3f41ee36302> > > > >::_Go() [c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.14.26428\include\thr\xthread : 230 + 0x110]
		ip=0xf97ffec sp=0x33bf9ec bp=0x33bfa3c 
	[8] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!std::_Pad::_Call_func(void *) [c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.15.26726\include\thr\xthread : 208 + 0x7]
		ip=0xf61372a sp=0x33bfa44 bp=0x33bfa44 
	[9] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!thread_start<unsigned int (__stdcall*)(void *)>(void * const) [d:\rs1\minkernel\crts\ucrt\src\appcrt\startup\thread.cpp : 115 + 0x2]
		ip=0xf967d15 sp=0x33bfa4c bp=0x33bfa80 
	[10] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0x33bfa88 bp=0x33bfa94 
	[11] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0x33bfa9c bp=0x33bfadc 
	[12] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0x33bfae4 bp=0x33bfaec 
	thread-6 tid=4408 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0x3cffe14 bp=0x3cffe88 
	[1] C:\Windows\System32\ntdll.dll!_NtWaitForSingleObject@12 + 0xa
		ip=0x76fb11ea sp=0x3cffe18 bp=0x3cffe88 
	[2] C:\Windows\System32\KERNELBASE.dll!WaitForSingleObjectEx + 0xb0
		ip=0x746b2770 sp=0x3cffe1c bp=0x3cffe88 
	[3] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!WASAPI_WaitDevice(SDL_AudioDevice *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\audio\wasapi\sdl_wasapi.c : 331 + 0x7]
		ip=0xf8d0b0b sp=0x3cffe90 bp=0x3addfb0 
	[4] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!SDL_RunAudio(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\audio\sdl_audio.c : 711 + 0x7]
		ip=0xf8a4ca6 sp=0x3cffeb0 bp=0x3addfb0 
	[5] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll + 0x3cb40
		ip=0xf57cb40 sp=0x3cffecc bp=0x3addfb0 
	[6] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!SDL_RunThread(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\thread\sdl_thread.c : 283 + 0x3]
		ip=0xf89c44e sp=0x3cffed0 bp=0x3addfb0 
		guessed frame
	[7] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!RunThreadViaBeginThreadEx(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\thread\windows\sdl_systhread.c : 105 + 0x11]
		ip=0xf8c3e17 sp=0x3cffef0 bp=0x3addfb0 
	[8] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll + 0x383e00
		ip=0xf8c3e00 sp=0x3cfff00 bp=0x3addfb0 
	[9] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!thread_start<unsigned int (__stdcall*)(void *)>(void * const) [d:\rs1\minkernel\crts\ucrt\src\appcrt\startup\thread.cpp : 115 + 0x2]
		ip=0xf967d15 sp=0x3cfff04 bp=0x3addfb0 
		guessed frame
	[10] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!_ceil_pentium4() + 0xdd
		ip=0xf967cbd sp=0x3cfff10 bp=0x3addfb0 
		guessed frame
	[11] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!_ceil_pentium4() + 0xdd
		ip=0xf967cbd sp=0x3cfff14 bp=0x3addfb0 
		guessed frame
	[12] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll + 0x40e030
		ip=0xf94e030 sp=0x3cfff30 bp=0x3cfff84 
		guessed frame
	[13] C:\Windows\System32\ntdll.dll!_wcstombs + 0x90
		ip=0x76fbabe0 sp=0x3cfff8c bp=0x3cfff9c 
	[14] C:\Windows\System32\ntdll.dll!FinalExceptionHandlerPad56 + 0x1
		ip=0x76fc71d9 sp=0x3cfffa4 bp=0xffffffff 
	[15] I:\OpenRCT2-0.2.1-develop-3aeb87b-windows-win32\openrct2.dll!_ceil_pentium4() + 0xdd
		ip=0xf967cbd sp=0x3cfffb0 bp=0xffffffff 
		guessed frame
	thread-7 tid=1952 
	[0] C:\Windows\System32\ntdll.dll!_KiFastSystemCallRet@0 + 0
		ip=0x76fb4dc0 sp=0xa3dfb3c bp=0xa3dfb7c 
	[1] C:\Windows\System32\ntdll.dll!_NtRemoveIoCompletion@20 + 0xa
		ip=0x76fb201a sp=0xa3dfb40 bp=0xa3dfb7c 
	[2] C:\Windows\System32\mswsock.dll + 0x1297b
		ip=0x7346297b sp=0xa3dfb44 bp=0xa3dfb7c 
	[3] C:\Windows\System32\kernel32.dll!@BaseThreadInitThunk@12 + 0x24
		ip=0x76d48f04 sp=0xa3dfb84 bp=0xa3dfb90 
	[4] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart + 0x2b
		ip=0x76f89c23 sp=0xa3dfb98 bp=0xa3dfbd8 
	[5] C:\Windows\System32\ntdll.dll!__RtlUserThreadStart@8 + 0x1b
		ip=0x76f89bf2 sp=0xa3dfbe0 bp=0xa3dfbe8 
```
https://github.com/OpenRCT2/OpenRCT2/blob/3aeb87b285b76677c90e8dd77cf5cef78668a847/src/openrct2/drawing/Drawing.String.cpp#L594

**Comment by janisozaur** [comment]
Fixed in #8533
